src/Text/Pandoc/Writers.hs

```diff
@@ -80,6 +80,7 @@ module Text.Pandoc.Writers
     , writeXWiki
     , writeZimWiki
     , writeVimdoc
+    , writeBBCode
```

Comment:
Presumably all the writer functions should be exported here (for the various BBCode variants), not just `writeBBCode`.